Assets/HoloToolkit/Common/Scripts/CrossPlatform/Reflection/ReflectionExtensions.cs

```diff
@@ -179,11 +179,12 @@ public static object[] GetCustomAttributes(this Type type, Type attributeType, b
 
     public static class ReflectionExtensions
     {
+#if !NET_4_6
         public static Type GetTypeInfo(this Type type)
```

Comment:
So, what do we use in place of this and does omitting it for standalone cause issue?

Reply:
The platform target shouldn't matter.